spec/controllers/personal_detail_controller_spec.rb

```diff
@@ -17,7 +17,7 @@
 
       expect(step.sex).to eq("male")
       expect(step.marital_status).to eq("Married")
-      expect(step.ssn).to eq("12345")
+      expect(step.ssn).to eq("123456789")
```

Comment:
I would also add a test for SSNs which start with a zero. I have been bitten by this with Boston-area zipcodes (e.g. 02215).